app/assets/javascripts/js-processor.js

```diff
@@ -24,6 +24,12 @@ import { transform as babelTransform } from "@babel/standalone";
 import { minify as terserMinify } from "terser";
 import RawHandlebars from "discourse-common/addon/lib/raw-handlebars";
 import { WidgetHbsCompiler } from "discourse-widget-hbs/lib/widget-hbs-compiler";
+import EmberThisFallback from "ember-this-fallback";
+
+const thisFallbackPlugin = EmberThisFallback._buildPlugin({
+  enableLogging: false,
+  isTheme: true,
+}).plugin;
```

Comment:
How about we do something like this?
```suggestion
const thisFallbackPlugin = EmberThisFallback._buildPlugin({
  enableLogging: false,
}).plugin;

delete thisFallbackPlugin.parallelBabel;
```

and then we can avoid the patch to `ember-this-fallback/index.js`?